campusmap/src/search-results/SearchResultsPage.test.tsx

```diff
@@ -0,0 +1,7 @@
+const sum = (a: number, b: number) => a + b;
+
+describe('<SearchResultsPage />', () => {
```

Comment:
is this just meant to be an example test case? I don't really see how this tests/relates to the Search Results Page directly.

Reply:
Yes, this is just an example to test that the test runner on GitHub actions works. The actual tests for the pages will be written soon.